platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -411,6 +411,26 @@ def _detect_sfp_type(self, sfp_type):
             # in this case we treat it as the default type according to the SKU
             self.sfp_type = sfp_type
 
+    def _is_qsfp_copper(self):
```

Comment:
PR title says CMIS, so where is CMIS copper cable check?

Reply:
The CMIS cable issue is not about copper or not, it is arbitrarily setting the temperature and voltage support statuses to true w/o reading the EEPROM, please check the PR description: "For CMIS cable, there is a bug in determining the DOM support status, the temperature and voltage support statuses are always set to true, this is not correct, they shall be determined in the same way as other DOM capability support statuses."